binzx/otomi

```diff
@@ -41,37 +41,73 @@ silent() {
     "$@" &>/dev/null
   fi
 }
+
+docker_tag_exists() {
+  curl -sflL https://index.docker.io/v1/repositories/$1/tags/$2 &>/dev/null
+}
```

Comment:
Are there any rate limits?

Reply:
Couldn't find any docs. And it is mostly from local machine. So hitting rate limit should be hard. But not definite